inc/winfsp/fsctl.h

```diff
@@ -250,7 +250,8 @@ enum
     UINT32 StreamInfoTimeout;           /* stream info timeout (millis); overrides FileInfoTimeout */\
     UINT32 EaTimeout;                   /* EA timeout (millis); overrides FileInfoTimeout */\
     UINT32 FsextControlCode;\
-    UINT32 Reserved32[1];\
+    UINT32 FlushOnCleanup:1;\
```

Comment:
It is not necessary to add this bit to `VolumeParams`. See my comments in the PR.